pyshelf/bulk_update/utils.py

```diff
@@ -23,24 +58,41 @@ def run(args):
     # a no-op and its easiest to use it in the subprocesses
     # because then it also automatically configures child
     # loggers such as boto and elasticsearch
-    handler = logging.StreamHandler(sys.stdout)
-    logger = logging.getLogger("update-search-index")
+    handler = _get_logging_handler(logger_name, args.get("--log-directory"))
```

Comment:
It is an optional parameter. But sounds like all of your comments are basically that this 2 scripts should be broken out from eachother. So this will all change